src/ui/PearlTheme.jsx

```diff
@@ -0,0 +1,150 @@
+import { createTheme } from '@mui/material';
+import '@fontsource/montserrat/800.css';
+import '@fontsource/montserrat/700.css';
+import '@fontsource/montserrat/600.css';
+import '@fontsource/montserrat/500.css';
+import { ThemeProvider } from '@mui/material';
+import CssBaseline from '@mui/material/CssBaseline';
+
+const fontFamily = 'Montserrat, sans-serif';
+
+const baseTheme = createTheme({
+  typography: {
+    fontFamily: fontFamily,
+    fontWeight: 600,
+    button: {
+      fontSize: '.875rem',
+      lineHeight: '1.125rem',
+      fontWeight: 600,
+      textTransform: 'none',
+    },
+    headingXL: {
+      fontSize: '3rem',
+      lineHeight: '3.5rem',
+    },
+    headingL: {
+      fontSize: '2.25rem',
+      lineHeight: '3rem',
+    },
+    headingM: {
+      fontSize: '1.75rem',
+      lineHeight: '2.25rem',
+    },
+    headingS: {
+      fontSize: '1.5rem',
+      lineHeight: '1.75rem',
+    },
+    xl: {
+      fontSize: '1.25rem',
+      lineHeight: '1.5rem',
+    },
+    l: {
+      fontSize: '1.125rem',
+      lineHeight: '1.375rem',
+    },
+    m: {
+      fontSize: '1rem',
+      lineHeight: '1.25rem',
+    },
+    s: {
+      fontSize: '.875rem',
+      lineHeight: '1.125rem',
+    },
+    xs: {
+      fontSize: '.75rem',
+      lineHeight: '1rem',
+    },
+  },
+  palette: {
```

Comment:
The Palette component can be removed if it's not going to be used for developing.